app/app/trade/[slab]/page.tsx

```
@@ -86,16 +121,45 @@ function TradePageInner({ slab }: { slab: string }) {
+        </div>
+      </div>
+
+      {/* ── DESKTOP: Full header ── */}
+      <div className="hidden lg:flex items-start justify-between px-4 py-2 gap-3 border-b border-[var(--border)]/30">
         <div className="min-w-0">
-          <p className="mb-1 text-[11px] font-medium uppercase tracking-widest text-[var(--accent)]">// TRADE</p>
-          <h1 className="text-2xl font-bold text-[var(--text)]" style={{ fontFamily: "var(--font-display)" }}>
-            {symbol}/USD <span className="text-base font-normal text-[var(--text-muted)]">PERP</span>
+          <p className="mb-0.5 text-[9px] font-medium uppercase tracking-[0.2em] text-[var(--accent)]/70">// TRADE</p>
+          <h1 className="text-lg font-bold text-[var(--text)]" style={{ fontFamily: "var(--font-display)" }}>
+            {symbol}/USD <span className="text-xs font-normal text-[var(--text-muted)]">PERP</span>
```

Biome (2.3.14)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)